sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/autorest.md

```diff
@@ -146,6 +149,25 @@ directive:
     $.parameters.push(stringIndexTypeParam);
     $.parameters.push(apiVersionParam);
 
+# Update descriptions to include a link to the REST API documentation.
+- from: analyzeconversations.json
+  where: $.paths.*.*
+  transform: |
+    var operationId = $.operationId.substring("ConversationAnalysis_".length);
+    // BUGBUG: Will not work until https://github.com/Azure/autorest.csharp/issues/2384 is fixed.
+    $["externalDocs"] = {
+        url: "https://docs.microsoft.com/rest/api/language/conversation-analysis-runtime/" + operationId.replace(/([a-z0�9])([A-Z])/g, "$1-$2").toLowerCase()
+    };
+
+- from: analyzeconversations-authoring.json
+  where: $.paths.*.*
+  transform: |
+    var operationId = $.operationId.substring("ConversationalAnalysisAuthoring_".length);
+    // BUGBUG: Will not work until https://github.com/Azure/autorest.csharp/issues/2384 is fixed.
```

Comment:
nit: still says bug, idk if it's fixed

Reply:
Yep - this is the ol' school way we tracked outstanding bugs in code. I just opened the linked bug, so definitely not fixed yet. :) I wanted to keep this for posterity, if nothing else. Hopefully one day *soon* it will work.